src/main/clm_instMod.F90

```diff
@@ -433,6 +438,14 @@ subroutine clm_instInit(bounds)
        call crop_inst%InitAccBuffer(bounds)
     end if
 
+    ! use_fates_spitfire is assigned an integer value in the namelist
+    ! see bld/namelist_files/namelist_definition_clm4_5.xml for details
+    if (use_fates_spitfire > 1) then
```

Comment:
And same here, you give it the generic type

Reply:
Same [comment](https://github.com/ESCOMP/CTSM/pull/991#discussion_r415424002).